lib/bootstrap_form/inputs/submit.rb

```diff
@@ -4,6 +4,10 @@ module BootstrapForm
   module Inputs
     module Submit
       def button(value=nil, options={}, &)
+        if options.key? :submits_with
+          options[:data] = { turbo_submits_with: setup_turbo_submit(options[:submits_with]) }
```

Comment:
I think this should `merge` with `options[;data]`, in case the user needs to specify other `data-` attributes.